### ISSUE TITLE
Test MPI request payloads in staging

### ISSUE THREAD

**Issue by ScottyRJames** [department-of-veterans-affairs/va.gov-team#55966]
## Issue Description
IAM team has said we can use any identifier to get profile from MPI. We need to do some testing to view what these requests look like and how we can use the Vista identifier to get the profile we need. 

---
## Tasks
- [x] Use test users identifiers in staging to get MPI payload
- [x] Record request and response of these payloads
- [x] Detail what a vista ID request might look like

## Acceptance Criteria
- [x] We know how to work with MPI data requests and responses
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by HeatherRienks** [comment]
@ScottyRJames - do you have an update on this ticket? Thanks!

**Comment by ScottyRJames** [comment]
@HeatherRienks No updates, haven't had a chance to test yet.

**Comment by ScottyRJames** [comment]
Unable to test in staging without ArgoCD access

Access ticket [template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=external-request%2Cplatform-tech-team-support%2Cops-access-request&template=vetsapi-argo-terminal-access.yaml&title=Vets-api+terminal+access+for+%5Bindividual%5D)

**Comment by HeatherRienks** [comment]
GH Ticket # TBD submitted by Scott for access.

**Comment by ScottyRJames** [comment]
Unblocking. ArgoCD access has been granted for dev and staging

**Comment by ScottyRJames** [comment]
Not attaching staging MPI info to public ticket but I've gathered the required info to move forward. Creating future work tickets next.

**Comment by HeatherRienks** [comment]
@ScottyRJames - can this be closed?

**Comment by HeatherRienks** [comment]
Per Scott, still awaiting review on a PR. He will ping the necessary team but it won’t deploy until 4/26 and we still need VBS to validate the changes.
